python-lib/tc_etl_lib/tc_etl_lib/auth.py

```diff
@@ -25,13 +25,13 @@
 
 import requests
 import logging
-from typing import Dict
+from typing import Dict, Optional
```

Comment:
```suggestion
from typing import Dict
```
Diría que no es necesario el "optional", se habrá quedado de alguna modificación anterior